sdk/core/azure-core/azure/core/settings.py

```diff
@@ -292,14 +304,14 @@ def set_value(self, value) -> None:
 
     def unset_value(self) -> None:
         """Unset the previous user value such that the priority is reset."""
-        self._user_value = _Unset
+        self._user_value = _unset
 
     @property
-    def env_var(self):
+    def env_var(self) -> Optional[str]:
         return self._env_var
 
     @property
-    def default(self):
+    def default(self) -> Union[Any, _Unset]:
```

Comment:
We are we using `Any` when we used `ValidInputType` for `default` in the constructor?

Reply:
Good catch, should be `ValidInputType`